### ISSUE TITLE
Allow to temporarily show individual passwords

### ISSUE THREAD

**Issue by redm123** [keepassxreboot/keepassxc#2266]
I'd like to have the possibility to quickly view an individual password in clear text.
Use case is usually that I want to manually enter a password somewhere, but are not sure which. I want to quickly check.

## Expected Behavior
Select a password entry, and with a single click show me the password.

## Current Behavior
Current options are:
* Show all in clear text - not what I want (apart from that it doesn't seem to work...)
* uncheck "Hide passwords in preview panel" - closer, but the problem is that I'd always have to manually close the panel, as otherwise one entry is always selected and the respective password is always visible...
    * the preview panel could be made autohide, but don't think that is what I want either, as it would also hide the notes for instance...
* open the edit dialog: double click the entry, go all the way to the right to hit the small eye icon, cancel - tedious

## Possible Solution
What I imagine for instance could be to show the eye button right next to the dotted password in the preview panel. Plus automatically hide the password again after 10s or so. (Actually, there is not really a use to show a couple of black dots in the preview panel ... ;) So they could be made useful by allowing to turn them into clear text)

KeePassXC - Version 2.3.1
Revision: 2fcaeea

Libraries:
- Qt 5.9.3
- libgcrypt 1.8.2

Operating system: macOS Sierra (10.12)
CPU architecture: x86_64
Kernel: darwin 16.6.0

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by kneitinger** [comment]
> Show all in clear text - not what I want (apart from that it doesn't seem to work...)

This option does work, but perhaps its poorly named. It means that in the entry's edit view, the passwords are cleartext; as if the eye icon was preclicked.
